app/Main.hs

```diff
@@ -2,12 +2,16 @@
 {-# LANGUAGE LambdaCase #-}
 {-# LANGUAGE OverloadedStrings #-}
 
+--
+-- Stackage and stdlib
+--
```

Comment:
Not really necessary imo, but I also don't really mind.

Reply:
Removed